src/web/resources/report.js

```diff
@@ -337,20 +337,29 @@ const MergedCostAccuracy = new Component('#pareto', {
         this.elt.replaceChild(this.plot(json["cost-accuracy"], json.tests.length), stub)
     },
 
-    plot: function(json, n) {
-        const sortLine = [...json[2]];
-        sortLine.sort((a, b) => {return a[0]-b[0]});
-        const ymax = 64 * n;
+    plot: function(speedAccuracy) {
+        const accuracyMax = speedAccuracy[0][1];
```

Comment:
Hmm, since we're already normalizing the cost axis, how about we normalize the accuracy axis on the backend as well? Then we wouldn't have this code here.